### PR TITLE
Use template strings for Slack message payload

### PR DESCRIPTION
**Pull request by yangshun**
Some of the Slack messages posted have a little bit of inconsistency and extra quotation marks:

screen shot 2017-01-11 at 10 47 07 pm

This PR converts them to template strings which is more readable. A node 6 upgrade is required also (presuming Heroku is being used for hosting).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,38 +7,36 @@ var app = express();
...
 app.post('/hook/v1/:tokens([\\/.\\w]*|)', gitbookMiddleware, function (req, res, next) {
     var messageBuilder = EVENTS[req.headers['x-gitbook-event']];
-
     if (!messageBuilder) {
         return res.status(200).end();
     }
 
     // Extract payload
-    var payload = req.body.payload;
+    var payload = req.body;
```
Comment (AaronO) [inline index.js]: Is there a reason why you changed this variable ?

**Review by AaronO — CHANGES_REQUESTED** [review]
Looks good as a whole, but you changed the extraction of the "payload" variable which might break things.

File: index.js (hunk 1 above)
Comment (yangshun) [reply index.js]: I'm not sure why `req.body.payload` works in the first place. Seems that `res.body` is the POST payload according to this: https://github.com/expressjs/body-parser#bodyparserjsonoptions.

My `body-parser` version is `1.14.1`.

Comment (AaronO) [reply index.js]: @yangshun Yep, `.payload` is an attribute of JSON delivered by the webhook, which looks like:

```json
{
    "eventName": "...",
    "payload": {
        ...
        "book": "org/book_id",
        ...
    }
}
```

Comment (yangshun) [reply index.js]: Ah I understand now. I was mistaken; I used this payload when testing locally.

```
{
  "book": { ... },
  "comment": { ... },
  "thread": { ... },
  "action": "opened"
}
```

Thanks for clarifying!

Comment (yangshun) [reply index.js]: Have fixed!

**Comment by yangshun** [comment]
Hi @AaronO I have fixed the `payload` issue. Could you have a look at this again?

**Comment by yangshun** [comment]
I have deployed my own version on Heroku and tested it. It works fine 👌

**Pull request closed without merging**